sql/updates/latest-dev.sql

```diff
@@ -1 +1,22 @@
 
+--add compression_state column and remove compressed column
+ALTER TABLE IF EXISTS _timescaledb_catalog.hypertable ADD COLUMN compression_state integer NOT NULL DEFAULT 0;
```

Comment:
To preserve correct column indexes, it is typically necessary to drop and recreate the whole table, including copying over data.

Otherwise there's a big risk the column indexes will be incorrect.

Reply:
@erimatnor This column (both new and old) don't have indexes. Do we still need to copy the data over?